### PR TITLE
Notebook doc updates

### PR DESCRIPTION
**Pull request by lintusj1**
#### Summary:
Update the documentation for version 0.6.1. The changes are mainly from rerunning the notebooks, although some manual changes have been made. 

Read the new docs from here: http://elfi.readthedocs.io/en/docs_proposals/

#### Copyright and Licensing

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD3 (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

File: docs/usage/parallelization.rst [hunk 1]
```diff
@@ -17,120 +17,142 @@ inference via different clients. Currently ELFI includes three clients:
...
 
+    elfi.set_client('multiprocessing')
 
+Any inference instance created after you have set the new client will
+automatically use it to perform the computations. Let's try it to our
+MA2 example model from the tutorial.
 
+When running the next command, take a look at the system monitor of your
+operating system; it should show 4 (or whatever number you gave the
+ipcluster start command) Python processes doing heavy computation
```
Comment (kskyten) [inline docs/usage/parallelization.rst]: I thought `ipcluster` is related to `ipyparallel` is it also needed with the `multiprocessing` cliient?

Comment (lintusj1) [inline docs/usage/parallelization.rst]: Good catch, that had slipped to the wrong place! You are right its only needed with ipyparallel.

File: docs/usage/parallelization.rst [hunk 2]
```diff
@@ -17,120 +17,139 @@ inference via different clients. Currently ELFI includes three clients:
...
+
+The multiprocessing client allows you to easily use the cores available
+in your computer. You can activate it simply by
 
+.. code:: python
 
+    elfi.set_client('multiprocessing')
 
+Any inference instance created after you have set the new client will
+automatically use it to perform the computations. Let's try it to our
```
Comment (kskyten) [inline docs/usage/parallelization.rst]: Let's try it with

File: docs/usage/tutorial.rst [hunk 3]
```diff
@@ -610,13 +670,43 @@ Now lets load all the parameters ``t1`` that were generated with numpy:
...
+    name = arraypool.name
+    print(name)
+
+
+.. parsed-literal::
+
+    arraypool_3615052699
+
 
+And open it up later to continue where we were left with it's name
```
Comment (kskyten) [inline docs/usage/tutorial.rst]: its name

File: docs/usage/tutorial.rst [hunk 4]
```diff
@@ -739,99 +829,109 @@ population. In essence, a population is just refined rejection sampling.
...
 It can be seen that the populations iteratively concentrate more and
 more around the true parameter values.
 
 Note that for the later populations some of the samples lie outside
 allowed region. This is due to the SMC algorithm sampling near previous
-samples, with *near* meaning a Gaussian distribution centered around
-previous samples with variance as twice the weighted empirical variance.
-However, the outliers carry zero weight, and have no effect on the
-estimates.
+samples. However, the outliers carry zero weight, and have no effect on
```
Comment (kskyten) [inline docs/usage/tutorial.rst]: However, the outliers carry zero weight and have no effect on...
(not an independent clause after "and" so no comma)

**Pull request merged**